project/Dependencies.scala

```diff
@@ -39,7 +39,8 @@ object Dependencies {
 
   val awsSdkS3            = "com.amazonaws"               % "aws-java-sdk-s3"          % "1.11.143"
 
-  val scalazStream        = "org.scalaz.stream"          %% "scalaz-stream"            % "0.8.6a"
+  val fs2Core             = "co.fs2"                     %% "fs2-core"                 % "0.10.3"
+  val fs2Io               = "co.fs2"                     %% "fs2-io"                   % "0.10.3"
```

Comment:
It looks like the cats dependency is transitive through `fs2`? Make sure that the version of `cats-effect` that it's pulling is at least `0.10` or the performance of the `IO` monad will be garbage. They recently improved it.

Perhaps consider adding an explicit dependency on `cats-effect` here?

Reply:
`cats-core`: 1.1.0
`cats-effect`: 0.10

Reply:
Good point to bump up cats versions manually as we plan to depend on it!